Packet++/header/PPPoELayer.h

```diff
@@ -361,23 +362,23 @@ namespace pcpp
 		/**
 		 * Get a PPPoE Tag by tag type.
 		 * @param[in] tagType The type of the tag to search
-		 * @return A PPPoETag object that contains the first tag that matches this type, or logical NULL
+		 * @return A PPPoETag object that contains the first tag that matches this type, or logical nullptr
```

Comment:
I would revert to `logical null` rather than `logical nullptr`. Ditto in the rest of the changes in this file